precompile/contract_native_minter.go

```diff
@@ -35,6 +36,7 @@ var (
 type ContractNativeMinterConfig struct {
 	AllowListConfig
 	UpgradeableConfig
+	InitialMint map[common.Address]*math.HexOrDecimal256 `json:"initialMint,omitempty"` // initial mint config to be immediately minted
```

Comment:
why are we using `*math.HexOrDecimal256` instead of `*big.Int` here?

Reply:
To make it similar with existing alloc mapping in genesis.